scripts/tasks/src/swc/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { type Options } from './types';
+
+const importPaths = {
+  es6: /from\s+["']([^"']+)["']/g,
```

Comment:
nit: we probably won't ever have these in Fluent but we might also want to support dynamic imports:

```js
import("file.js")
```

Reply:
yeah, at that stage IMO a proper AST would be necessary. this function is not used ATM ( i kept it only for future reference - will add TODO though with your suggestion ) ty!